### DISCUSSION TITLE
any way to check if the lobby owner has the anticheat mod?

### DISCUSSION THREAD

**Discussion by zx3800** [winstxnhdw/lc-hax#472] [Ideas]
is it possible for the client to check if the host has the mod enabled or something?

**Comment by winstxnhdw** [comment]
Don't usually these lobbies already have `[AC]` in their names?

**Reply by arandomuser0** [reply]
you can remove that prefix at the config

**Reply by winstxnhdw** [reply]
Why would a host do that?

**Reply by winstxnhdw** [reply]
Anyways, the answer is maybe but I am not going to invest time into figuring out what unique feature of the host I should be looking for.
